src/cosalib/cli.py

```diff
@@ -0,0 +1,119 @@
+# NOTE: PYTHONUNBUFFERED is set in cmdlib.sh for unbuffered output
+# pylint: disable=C0103
+
+import argparse
+import logging as log
+import os
+
+#: All valid logger levels
+LOG_LEVELS = ('warn', 'error', 'debug', 'info')
+
+
+class Cli:
```

Comment:
I think I should switch this to a subclass of `argparse.ArgumentParser`

Reply:
I like that idea.

Reply:
Done